index.js

```diff
@@ -304,3 +305,5 @@ module.exports = function(configObject) {
     }
   }
 }
+
+module.exports.payload = payload
```

Comment:
ok, now I get what you were trying to do.
if you're trying to make this visible to consumers (us), then yes, I would say external library. However, I would rather copy the file in two places.